comp/core/gui/guiimpl/gui.go

```diff
@@ -155,7 +157,8 @@ func (g *gui) start(_ context.Context) error {
 
 	g.listener, e = net.Listen("tcp", "127.0.0.1:"+g.port)
 	if e != nil {
-		return e
+		g.logger.Errorf("GUI server didn't achieved to start: ", e)
```

Comment:
As `start` never returns an error you can update the signature to `func (g *gui) start(_ context.Context)`.